python/cudf/tests/test_pandas_interop.py

```diff
@@ -8,8 +8,9 @@
 
 def test_to_pandas():
     df = DataFrame()
-    df['a'] = np.arange(10, dtype=np.int32)
-    df['b'] = np.arange(10, 20, dtype=np.float64)
+    df['a'] = np.arange(5, dtype=np.int32)
+    df['b'] = np.arange(10, 15, dtype=np.float64)
+    df['c'] = np.array([True, False, None, True, True])
```

Comment:
You aren't testing this case in the below tests.

Reply:
Oops, fixed in cb928ad